client/src/components/Pages/Home/index.js

```diff
@@ -1,36 +1,90 @@
 import React from 'react';
 
-import EmptyView from '../../Common/EmptyView';
+// import EmptyView from '../../Common/EmptyView';
+import LanguageItem from '../../Common/LanguageItem';
+import HobbyItem from '../../Common/HobbyItem';
+import { getAgeByBirthdate } from '../../../utils/common';
+import { myProfilePicture } from '../../../images';
+import {
+  // emptyViewDescription,
+  aboutMeDescription,
+  languages, hobbies,
+} from './constants';
+
+import './styles.css';
 
 const Home = () => {
-  const renderDescription = () => (
+  const renderText = array => (
+    array.map(item => <p key={item}>{item}</p>)
+  );
+
+  const renderLanguages = () => (
+    languages.map(item => <LanguageItem {...item} key={item.id} />)
+  );
+
+  const renderHobbies = () => (
+    hobbies.map(item => <HobbyItem {...item} key={item.id} />)
+  );
+
+  const renderBasicInfo = () => (
+    <div className="basic-info-section section row">
+      <div className="left-column col s12 m6">
+        <img src={myProfilePicture} alt="Me" className="my-profile-photo" />
+      </div>
+      <div className="right-column col s12 m6">
+        <h4 className="name-lastname">Dmytro Kovalenko</h4>
+        <p className="job-title">Full Stack JS Developer</p>
+        <p className="location">
+          <a href="https://www.google.es/maps?q=41.382207,2.140318" target="_blank" rel="noopener noreferrer">
+            Sants Estació, Barcelona, Spain
+          </a>
+        </p>
+        <p>
+          <strong>Contact info: </strong>
+          Contact me in
+          {' '}
+          <a href="https://www.linkedin.com/in/kovalenkodmytro/" target="_blank" rel="noopener noreferrer">
+            LinkedIn
+          </a>
+        </p>
+        <p><strong>Age: </strong>{getAgeByBirthdate('1992-05-20')} years old</p>
+        <p><strong>Civil status: </strong>Married, no children</p>
+      </div>
+    </div>
+  );
+
+  const renderInfo = () => (
     <React.Fragment>
-      <p>
-        Welcome to my new personal project!!!
-      </p>
-
-      <p>
-        Ask questions as you are speaking to me and try to populate
-        the website with the info he will provide.
-      </p>
-
-      <p>
-        Every page will have a progress bar which indicates
-        how much info you have already recovered - so you will know
-        where to stop asking about some particular topic.
-      </p>
-
-      <p>
-        One more time, this project was created to learn about Chatbots
-        and to practice my full stack skills. I am happy at my current job and
-        I am not looking for a change.
-      </p>
+      {renderBasicInfo()}
```

Comment:
It was a great improvement to encapsulate all these paragraphs into one single function. Great job! 😃